frontend/src/components/Buttons/SortButton.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-btn @click="changeOrder">
+  <v-btn icon @click="changeOrder">
```

Comment:
```suggestion
  <v-btn icon @click="changeOrder" v-bind="props">
```

This would fix the sort button not being disabled on page load, like in master. Not strictly related to this PR but a good to have fix anyway.